src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const totalPop = document.getElementsByClassName('total-population');
+
+const popul = Array.from(document.getElementsByClassName('population'))
+  .map((pop) => Number(pop.innerText.replace(/,/g, '')));
+
+Array.from(totalPop)[0].textContent = Number(popul
+  .reduce((total, amount) => total + amount))
+  .toLocaleString('en');
```

Comment:
```suggestion
Array.from(totalPop)[0].textContent = Number(
  popul.reduce((total, amount) => total + amount)
).toLocaleString('en');
```
Записывай так, если надо переносить строку

Reply:
Привет! 
Обрати внимание на стиль кода, [который ты мне советовал](https://github.com/mate-academy/js_style-to-object/pull/526#discussion_r608844760) и я перенял. Строка начинается с метода.

Reply:
Этот стиль работает тогда когда ты не передаешь результат метода в другую функцию(тут у тебя результат reduce попадает в `Number()`).